dvc/progress.py

```diff
@@ -103,7 +103,7 @@ def _bar(self, target_name, current, total):
         n_sp = bar_len - n_sh
         pbar = "[" + "#" * n_sh + " " * n_sp + "] "
 
-        return num + pbar + percent + target_name
+        return num + pbar + percent + str(target_name)
```

Comment:
This feels like a coverup that would make us not notice bugs :slightly_smiling_face:

Reply:
It is common to pass paths here, do you mean that we should enforce passing strings only? We may add:
```python
assert isinstance(target_name, (str, PurePath)
```
To fail if something completely irrelevant is passed.